lib/ansible/modules/windows/win_dns_client.py

```diff
@@ -19,15 +19,14 @@
   adapter_names:
     description:
       - Adapter name or list of adapter names for which to manage DNS settings ('*' is supported as a wildcard value).
-      - The adapter name used is the connection caption in the Network Control Panel or via C(Get-NetAdapter), eg C(Local Area Connection).
-    type: str
```

Comment:
Why remove the type?

Reply:
Also a list is allowed and only one type can be specified.

Reply:
set the type to list then